src/assets/translations/en/main.json

```diff
@@ -341,6 +341,9 @@
         "unstake": "Unstake",
         "instantUnstake": "Instant Unstake",
         "claimWithdraw": "Claim Withdraw"
+      },
+      "erc20Approve": {
+        "approve": "Approve %{symbol}"
```

Comment:
most of the complexity in this PR can be avoided by not translating a symbol - we can't translate a symbol anyway, so let's just translate `Approve` using some common translation and show the symbol next to it?